css-three-column-layout/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="header letter-spacing text-align-center">
+      <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+    </div>
+    <div class="row img-row">
+      <div class="column-one-third white-background">
+        <img src="images/merge.png" alt="Streamlined QMS" class="icon-image">
+      </div>
+      <div class="column-one-third white-background">
+        <img src="images/cloud-computing.png" alt="Integrates with existing tools" class="icon-image">
+      </div>
+      <div class="column-one-third white-background">
+        <img src="images/discussion.png" alt="In-depth Onboarding" class="icon-image">
+      </div>
+    </div>
+    <div class="row text-row">
+      <div class="column-one-third">
+        <h3 class="letter-spacing text-align-center">Streamlined QMS</h3>
+        <p class="text-align-center">A QMS that includes modules for all stages of product life cycle from Design Control to CAPA</p>
+      </div>
+      <div class="column-one-third">
+        <h3 class="letter-spacing text-align-center">Integrates with existing tools</h3>
+        <p class="text-align-center">Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+      </div>
+      <div class="column-one-third">
+        <h3 class="letter-spacing text-align-center">In-depth Onboarding</h3>
+        <p class="text-align-center">We train your entire team and provide you with support when and as it's needed</p>
+      </div>
+    </div>
```

Comment:
This is still overly separated. The image and text are related, so they should all be grouped together. The image, heading, and text are all related, so they should all be children of one column class

Reply:
<img width="1046" alt="Screen Shot 2021-10-14 at 5 37 59 PM" src="https://user-images.githubusercontent.com/72685552/137414311-1dc81824-9b27-4765-9da1-a9303117b196.png">